lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +52,12 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        for i in text:
+            if i in self._punctuation:
+                text = text.replace(i, ' ')
+        splited_text = text.split()
+        tokens = [token.lower() for token in splited_text]
```

Comment:
It's better to call `.lower()` for all `text` before `split`

Reply:
fixed